static/scenes/railroad/index.html

```diff
@@ -34,7 +34,28 @@
 
 <div id="content"></div>
 
-<script src="index.js" type="module"></script>
+<script type="module">
+  import api from '../../src/scene/api.js';
+  import Game from './:closure.js';
+  
+  api.ready(() => initialize());
+  
+  api.config({
+      pause: true,
+      orientation: 'landscape',
+      sound: ['music_start_scene'],
+    });
+  
+  function initialize() {
+    const contentElement = document.querySelector('#content');
+  
+    const game = new Game();
+    api.addEventListener('pause', (ev) => game.pause());
+    api.addEventListener('resume', (ev) => game.resume());
+    api.addEventListener('restart', (ev) => game.restart());
+    game.start(contentElement);
+  }
```

Comment:
I think this should still be able to be in index.js

Reply:
Done. For some reason when I was testing it seemed like the magic "./:closure.js" url import only worked in index.html. Must have been something else I was doing wrong.